dialects/postgresql/src/main/kotlin/app/cash/sqldelight/dialects/postgresql/grammar/PostgreSql.bnf

```diff
@@ -198,6 +200,21 @@ string_literal ::= string {
   override = true
 }
 
+literal_value ::= ( {numeric_literal}
+                  | {string_literal}
+                  | {blob_literal}
+                  | NULL
+                  | CURRENT_TIME
+                  | CURRENT_DATE
+                  | CURRENT_TIMESTAMP
+                  | interval_expression ) {
```

Comment:
Override literal_value - match literal expression for example `INTERVAL '1 day'`